### ISSUE TITLE
6 images can't be downloaded 

### ISSUE THREAD

**Issue by copperfield42** [magefree/mage#8082]
MAGE client version 1.4.50-V1-beta2 (build: 2021-07-30 08:39) 
```
INFO  2021-07-31 10:03:20,654 Started download of 2 images from source: scryfall.com, language: en, threads: 3           =>[Thread-1315] DownloadPicturesService.run 
WARN  2021-07-31 10:03:23,200 Image download for Shorecomber Crab (OANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/oana/20/en?format=image =>[pool-10-thread-1] DownloadPicturesService$DownloadTask.run 
WARN  2021-07-31 10:03:23,201 Image download for Shorecomber Crab (OANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/oana/20/?format=image =>[pool-10-thread-1] DownloadPicturesService$DownloadTask.run 
WARN  2021-07-31 10:03:24,633 Image download for Forest (ANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/ana/20a/en?format=image =>[pool-10-thread-2] DownloadPicturesService$DownloadTask.run 
WARN  2021-07-31 10:03:24,633 Image download for Forest (ANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/ana/20a/?format=image =>[pool-10-thread-2] DownloadPicturesService$DownloadTask.run 
INFO  2021-07-31 10:03:28,125 Images: search broken files...                                                             =>[AWT-EventQueue-0] DownloadPicturesService.startDownload 
```

the [Shorecomber Crab](https://scryfall.com/card/xana/20/shorecomber-crab) is from XANA and the [forest](https://scryfall.com/card/ana/20/forest) code number isn't 20a, it just 20

**Comment by copperfield42** [comment]
update from today new beta: 1.4.50-dev (2021-08-06 17-45)
```
                                                                          =>[SwingWorker-pool-6-thread-3] SessionImpl$4.work 
INFO  2021-08-06 14:58:03,439 Started download of 23 images from source: scryfall.com, language: en, threads: 3          =>[Thread-1443] DownloadPicturesService.run 
WARN  2021-08-06 14:58:04,635 Image download for Shorecomber Crab (OANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/oana/20/en?format=image =>[pool-9-thread-1] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:04,635 Image download for Shorecomber Crab (OANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/oana/20/?format=image =>[pool-9-thread-1] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:04,657 Image download for Bounty of the Deep (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/7/en?format=image =>[pool-9-thread-3] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:04,658 Image download for Bounty of the Deep (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/7/?format=image =>[pool-9-thread-3] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:04,843 Image download for Forest (ANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/ana/20a/en?format=image =>[pool-9-thread-2] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:04,843 Image download for Forest (ANA) failed - responseCode: 404 url: https://api.scryfall.com/cards/ana/20a/?format=image =>[pool-9-thread-2] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:05,575 Image download for Faceless Agent (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/31/en?format=image =>[pool-9-thread-1] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:05,575 Image download for Faceless Agent (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/31/?format=image =>[pool-9-thread-1] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:05,715 Image download for Manor Guardian (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/16/en?format=image =>[pool-9-thread-3] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:05,715 Image download for Manor Guardian (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/16/?format=image =>[pool-9-thread-3] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:05,793 Image download for Skyshroud Lookout (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/29/en?format=image =>[pool-9-thread-2] DownloadPicturesService$DownloadTask.run 
WARN  2021-08-06 14:58:05,794 Image download for Skyshroud Lookout (J21) failed - responseCode: 404 url: https://api.scryfall.com/cards/j21/29/?format=image =>[pool-9-thread-2] DownloadPicturesService$DownloadTask.run 
INFO  2021-08-06 15:01:02,627 Images: search broken files...                                                             =>[AWT-EventQueue-0] DownloadPicturesService.startDownload 

```

**Comment by JayDi85** [comment]
ANA* and OGW images fixed in f3e38f1026b4fa462401477d49a89c98cbf5feed. J21 images will be fixed after scryfall update (99a561b136712ef5c3d0c142fabb3d98dd687a89).
